codebuild/scripts/release_tag.sh

```diff
@@ -57,5 +81,9 @@ for row in $(echo ${ACCOUNTS_ESCAPED} | jq -r '.[] | @base64'); do
   retag_image "$image" "$old_tag" "$minor_tag"
   retag_image "$image" "$old_tag" "$major_tag"
```

Comment:
Is `latest` tag automatically applied when pushing a new image? Will it be a problem if the corresponding smlogs binary is not tagged with `latest`?

Reply:
Latest is always applied for both the images and the binaries upon running the deployment step. This step only handles the case where we want to apply a specific tag to that (or another) commit.